exchanges/btse/btse.go

```diff
@@ -615,3 +615,24 @@ func (b *BTSE) calculateTradingFee(ctx context.Context, feeBuilder *exchange.Fee
 func parseOrderTime(timeStr string) (time.Time, error) {
 	return time.Parse(time.DateTime, timeStr)
 }
+
+// MillionPairs returns a map of symbol names which have a IsMillion equivalent
+func (m *MarketSummary) MillionPairs() map[string]struct{} {
+	pairs := map[string]struct{}{}
+	for _, s := range *m {
+		if s.Active && s.HasLiquidity() && s.IsMillions() {
+			pairs[strings.TrimPrefix(s.Symbol, "M_")] = struct{}{}
+		}
+	}
+	return pairs
+}
+
+// HasLiquidity returns if a market pair has a bid or ask != 0
+func (m *MarketPair) HasLiquidity() bool {
+	return m.LowestAsk != 0 && m.HighestBid != 0
```

Comment:
You say `or` in the comment but require both Ask and Bid to be populated

Reply:
Thank you

Fixed e7e5b13f